iModelCore/BeSQLite/PublicAPI/BeSQLite/BeSQLite.h

```diff
@@ -2424,6 +2381,7 @@ struct DbFile : NonCopyableClass
     Savepoint m_defaultTxn;
     BeBriefcaseId m_briefcaseId;
     StatementCache m_statements;
+    mutable NoCaseCollation m_noCaseCollation;

```

Comment:
```suggestion
    NoCaseCollation m_noCaseCollation;
```
Now that `SetNoCaseCollation` is no longer `const`.